skimage/transform/radon_transform.py

```diff
@@ -34,13 +40,19 @@ def radon(image, theta=None, circle=True, *, preserve_range=False):
         Whether to keep the original range of values. Otherwise, the input
         image is converted according to the conventions of `img_as_float`.
         Also see https://scikit-image.org/docs/dev/user_guide/data_types.html
+    center : array_like, optional
+        Coordinates of the rotation axis. If None, the rotation axis will be
+        located in the pixel with coordinates:
+        ``(image.shape[0] // 2, image.shape[1] // 2)``
+        This will change in a future release, and it will be moved to:
+        ``((image.shape[0] - 1) / 2, (image.shape[1] - 1) / 2)``
 
     Returns
     -------
     radon_image : ndarray
         Radon transform (sinogram).  The tomography rotation axis will lie
-        at the pixel index ``radon_image.shape[0] // 2`` along the 0th
-        dimension of ``radon_image``.
+        by default at the position ``radon_image.shape[0] // 2`` along the 0th
+        dimension of ``radon_image``. This will change in a future release.
```

Comment:
```suggestion
        axis of ``radon_image``. This will change in a future release.
```
"In NumPy dimensions are called axes." ([ref](https://numpy.org/devdocs/user/quickstart.html#the-basics)).

Reply:
But I admit that, in this particular case, this may cause confusion with the *rotation axis* we keep talking about...